docs/faq.md

```diff
@@ -2,3 +2,137 @@
+Scalar markets can also be used as leverage if you tighten the range that’s being traded.
+
+Combinatorial Prediction Market:
+A combinatorial prediction market allows traders to place wagers on compositions of interconnected outcomes. For example, let’s take a scenario in which a public company wants to use a prediction market to get a better idea of how a particular strategy impacts their market capitalization. They could make a combinatorial wager on the price of their shares after a certain date, composed of whether or not a particular strategy is implemented. Combinatorial markets are ideal for aggregating precise information.
+
+
+Q: How does the Zeitgeist network determine the outcome of an event?
+
+A: When a new prediction market is created. The creator of that prediction market will select an oracle to provide the true conclusion of the event. The creator must bond some ZTG tokens to the oracle which will be slashed if the Oracle is found to report a false outcome.
+
+Q:What are the limitations of centralized prediction markets in comparison to decentralized prediction markets?
+
+A: One example of the difference between centralized and decentralized prediction markets is the ability of token holders to sway the outcome of a network-upgrade proposal via the futarchy model of on-chain governance. 
+
+Q: Difference from other decentralized prediction markets?
+
+A: Zeitgeist is very different from other decentralized prediction market platforms.
+We are building an open and evolving protocol to support prediction markets primitives, including the governance for how this protocol changes over time. This means we are also building an ecosystem for prediction markets and applications based on prediction markets.
+
+We will have a SDK so developers who want to build their pm-based app can plug into our tech stack, and build their applications on the Zeitgeist protocol. We are building applications ourselves too. To top it off we'll be marketing and supporting prediction markets.
+ We also will be supporting select external teams with grants through the Zeitgeist Foundation, a non-profit entity separated from ZeitgeistPM LLC, in a mission to foster healthy growth throughout the Zeitgeist network.
+
+
+unanswered
+==========
+
+Q: How will Zeitgeist try to get enough people to use it, so prediction markets are valid enough and don't suffer from illiquidity? - prediction markets
+
+Q: Where can I find Zeitgeist Documentation? - development
```

Comment:
Which documentation are we referring here?

FAQs are going to be part of the documentation right.